packages/vaadin-dialog/src/vaadin-dialog.d.ts

```diff
@@ -77,7 +77,7 @@ declare class DialogElement extends ThemePropertyMixin(
    * default) means that the `aria-label` attribute is not present at
    * all.
    */
-  ariaLabel: string | null | undefined;
+  ariaLabel: string;
```

Comment:
Should we update the default value for the `ariaLabel` property as well (to fill it with an empty string by default)?

Reply:
Good point, done.